skimage/restoration/_denoise_cy.pyx

```diff
@@ -277,5 +205,3 @@ def _denoise_tv_bregman(image, double weight, int max_iter, double eps,
 
         rmse = sqrt(rmse / total)
         i += 1
-
-    return np.squeeze(np.asarray(u[1:u_height-1, 1:u_width-1]))
```

Comment:
Is it important that this return statement no longer exists here? I assume that's because `out` is handed in above?